### DISCUSSION TITLE
【水贴】感觉现在绳匠不太好干了

### DISCUSSION THREAD

**Discussion by maomiaonei** [share121/inter-knot#951] [其他]
如题，以前以为绳匠来丁尼来的快，现在感觉好难。之前用的萝卜数据还没了，新人绳匠又不知道从哪里搞来萝卜。开始怀疑自己能不能干好绳匠了，听说有些委托人能讨要到萝卜，有没有资深绳匠帮忙引荐一个。

**Comment by ran-mitake-ag** [comment]
1500丁尼从空洞到提瓦特一条龙

**Comment by kallen1314** [comment]
我推荐HIA俱乐部，有面向新人的活动哦，奖励也不错！

**Reply by gaojiaobei** [reply]
byd骗人自首是吧，HIA是官方的俱乐部啊！不要糊涂把自己送进去啊

**Comment by gaojiaobei** [comment]
毕竟不是谁都有传奇绳匠【法厄同】那样的实力，绳匠这行人也多，还是灰色产业，没活是正常的。
并且就连法厄同这样的传奇人物都被抓了，绳匠这行是危险，钱来还要靠运气，没有特别出色的制造萝卜的能力或渠道，绳匠这行没得干。

**Comment by 1451374815** [comment]
大坏境不好，业务又卷，难搞啊

**Comment by Lanxey** [comment]
僧多粥少啊，委托就那么点，绳匠一大堆，卷的要死，特别是这俩月莫名多了一堆小号开脚本抢委托，抢了还不做，真是脑子被以骸吃了。

**Reply by 1451374815** [reply]
这个委托我接了

**Comment by Larksonguesinaspic** [comment]
3楼搞笑呢？
我前两天还看见他俩在六分街瞎逛
